package spob;

import "spob/Entry.proto";

message RecoverPropose {
  required uint32 primary = 1;
  enum RecoverType {
    DIFF = 1;
    TRUNC = 2;
  }
  required RecoverType type = 2;
  repeated Entry proposals = 3; // for DIFF
  optional uint64 last_mid = 4; // for TRUNC
}